src/components/order-service/order-service-bind.html

```diff
@@ -3,6 +3,7 @@
                    applications="$ctrl.applications"
                    form-name="$ctrl.forms.bindForm"
                    allow-no-binding="true"
+                   project-name="$ctrl.projectDisplayName"
```

Comment:
we probably should have named the variable project-display-name so that someone doesn't try to use it later as the unique name for something, can you open an issue to fix that later?